File: src/test/samples/sample.go
package main

import "fmt"

// #region FirstRegion
var x = 42
// #endregion

// #region SecondRegion
type MyClass struct{}

// #region InnerRegion
func (m MyClass) myMethod() {}
// #endregion

// #endregion

func main() {
    fmt.Println("Sample")
}

File: src/test/samples/sample.fs
//#region   FirstRegion
let x = 42
//#endregion

// #region SecondRegion
type MyClass() =
    // #region InnerRegion
    member this.MyMethod() = ()
    // #endregion ends InnerRegion

    // #region
    member this.MyOtherMethod() = ()
    //        #endregion
// #endregion

